app/code/Magento/Catalog/view/frontend/templates/product/view/addto.phtml

```diff
@@ -6,7 +6,11 @@
 
 // @codingStandardsIgnoreFile
 
-/** @var $block \Magento\Catalog\Block\Product\View*/
+/**
+ * Product view template
```

Comment:
Does this comment bring any value? Template file name itself is more specific and `templates/product/view` path already contains this information.

Same applicable to https://github.com/magento/magento2/blob/develop/app/code/Magento/Catalog/view/frontend/templates/product/view/form.phtml#L10